qchem/pennylane_qchem/qchem/__init__.py

```diff
@@ -14,6 +14,7 @@
 """The PennyLane quantum chemistry package. Supports OpenFermion, PySCF,
 and Psi4 for quantum chemistry calculations using PennyLane."""
 from .structure import *
+from .structure import force_constants

```

Comment:
I had to import ``force_constants`` explicitly here to avoid CodeFactor to complain. But this shouldn't be necessary, right?. I wonder why ``from .structure import *`` is not working for the new function here.

Reply:
I think the message was:

```
Undefined variable name 'force_constants' in __all__ (undefined-all-variable)
```